### PR TITLE
Sdesen docs

### PR DESCRIPTION
**Pull request by sdesen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,3 +225,49 @@ yarn dlx prisma migrate dev --name <some nice description of the changes you mad
...
+1. The **Requesting Application** is the Client application which will make the token exchange request to the IdP, ultimately requesting protected resource access from the Resource Server. 
+
+1. The **Resource Application** is the application which owns the protected resources and must issue access tokens to the Requesting Application using the flow.
+
+You app can support this flow as a **Requesting App**, a **Resource App**, or both! We recommend integrating in only on way to get started. The section will also cover best practices based on the architecture of the given apps.
+
+## Requesting App Steps
+The Requesting App is the application that is requesting or querying objects or pulling data from another application. This section describes the specific steps to build the ID Assertion Authorization Grant into the application. The Requesting App must make a separate request to IdP for an `ID-JAG` token for each unique resource the user wants to access. To prevent performance issues, follow these best practices:
+
+> **Tip!**  Checkout the Debug Console in the App for detailed views of the requests!
```
Comment (kamronbatman) [inline README.md:245]:
```suggestion
> [!TIP]
> Checkout the Debug Console in the App for detailed views of the requests!
```

File: README.md [hunk 2]
```diff
@@ -289,3 +225,49 @@ yarn dlx prisma migrate dev --name <some nice description of the changes you mad
...
+ 1. **Only request when needed**. JAG tokens should be requested at the last responsible moment. JAG tokens are not meant to be long lived. Loading does not need to be bulk loaded and optimized. Trying to optimize the requests for multiple JAG tokens can result in poor performance at scale.
+ 1. **Do not store JAG Tokens. Store ID, Access and Refresh Tokens**. JAGs are really short lived. As mentioned earlier, JAGs are passed from Requesting Apps to Resource Apps to exchange for an Access Token to access the Resource App. The Access Token is the valuable artifact to be stored and protected. The JAG token is effectively useless once exchanged, so it should be discarded.
+
+
+## Resource App Steps
+
+The Resource App is the application that owns protected resources normally accessed via OAuth2.0 by Requesting App users. This section describes the specific steps to support the ID Assertion Authorization Grant into the application. The Resource App is responsible for validating  `ID-JAG` tokens before issuing OAuth `access_tokens` as it normally would.
+
+> **Tip!**  Checkout the Debug Console in the App for detailed views of the requests!
```
Comment (kamronbatman) [inline README.md:268]:
```suggestion
> [!TIP]
> Checkout the Debug Console in the App for detailed views of the requests!
```

**Review by kamronbatman — CHANGES_REQUESTED** [review]
Looks great, just some style nits!

**Comment by jefftaylor-okta** [comment]
I just reviewed and these docs look awesome!

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**